test/factories/surveys.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :survey do
+    difficulty { 1 }
+    realism { 1 }
+    interest { 1 }
+    comment { "MyText" }
+    submitted_flag_id { 1 }
```

Comment:
Using Faker we can get a random number from 1-5 here every time instead of just testing with 1 every time.